packages/modules/src/document/Document.ts

```diff
@@ -35,7 +35,7 @@ import * as Did from '@cord.network/did'
 import { blake2AsHex } from '@polkadot/util-crypto'
 import { ConfigService } from '@cord.network/config'
 
-function getHashRoot(leaves: Uint8Array[]): Uint8Array {
+export function getHashRoot(leaves: Uint8Array[]): Uint8Array {
```

Comment:
why?

Reply:
It was necessary because I had been using multiple hashes to create the identifier. However, based on your previous comment, I have now updated the code to utilize a single hash, so this is no longer necessary. I will make the change accordingly.